script/Ciel/Day 14/14_01B_03_CIEL14B_4_1_2.txt

```diff
@@ -532,20 +589,21 @@ But it doesn't matter.
 [sha:db898a47e3f522b62240f3773722677f383b52c6]{
 -- Page 168, Offset 40745.
 -- あと少し。あと数歩で学校だ。
-Just a little more. In a few more steps, she will reach the school.
+She's almost there now.
+ With just a few more steps, she will reach the school.
 }
 [sha:171fb5126d1854d8a5e9a1d1719c6f5b8640916e]{
 -- Page 168, Offset 40746.
 -- もう豪快に踏み潰してやるんだから、と強がりを口にする。
-'I will trample down on it with all my heart', she says with strong demeanor.
+She tells herself that she's going to absolutely trample it when she gets there.
 }
 [sha:51401794c685b7d7b062206cac7d514f013065ea]{
 -- Page 169, Offset 40747.
 -- でもどうせなら。
-But if that's how it was going to be...
+But if she was going to vanish either way...
 }
 [sha:1f8f1722bad485792a0c2f1b13a6e91bb33986b3]{
 -- Page 169, Offset 40748.
 -- あの時、志貴に拒絶された時に、消えてしまえればよかったのに、と彼女は思った。
-She thought to herself that it would have been better if she had vanished the moment Shiki had rejected her.
+She wished she could have vanished the moment Shiki had rejected her instead.
```

Comment:
conjunction probably not needed here, it muddles things up
>If she was going to vanish either way———

tense and some small tweaks
>Then she would rather have vanished the moment Shiki rejected her instead.